README.md

```diff
@@ -0,0 +1 @@
+Check the READMe.md files in the client and server folders for appropriate guidelines.
```

Comment:
Please clarify how you want me to review the README file. Should I develop the content further or is there any other action you want me to take?

Reply:
The only thing I want you to do is approve the changes. If there are more contents you'd want me to add, please tell me, or if you wanna do it yourself, checkout to my branch `Don-christo-patch-1` and make your desired changes there, then pull and push. If there are none, please approve the PR. I'll re-request your review so you have the option to approve the code again.

Reply:
Okay.